python/ray/autoscaler/_private/load_metrics.py

```diff
@@ -10,7 +10,7 @@
 from ray.autoscaler._private.constants import MEMORY_RESOURCE_UNIT_BYTES,\
     AUTOSCALER_MAX_RESOURCE_DEMAND_VECTOR_SIZE
 from ray.autoscaler._private.util import add_resources, freq_of_dicts
-from ray.gcs_utils import PlacementGroupTableData
+from ray._private.gcs_utils import PlacementGroupTableData
```

Comment:
@wuisawesome Would this impact any backward compatibility of monitor?

Reply:
Hmmm good catch. Let me investigate before we merge.

Reply:
I don't think that this would? Old autoscalers will use the old version of the Ray lib with `ray.gcs_utils` still being valid, and new autoscalers will use the new version of the Ray lib with `ray._private.gcs_utils` being valid. Nothing should change about the cluster <--> autoscaler interaction.

Lmk if I'm missing something there.

Reply:
ok, this should be fine.